aten/src/ATen/native/BinaryOps.cpp

```diff
@@ -120,6 +121,21 @@ static inline void sub_check(const Tensor& self, const Tensor& other) {
               "If you are trying to invert a mask, use the `~` or `logical_not()` operator instead.");
 }
 
+Tensor& remainder_out(Tensor& result, const Tensor& self, const Tensor& other) {
+  auto iter = TensorIterator::binary_op(result, self, other);
+  remainder_stub(iter.device_type(), iter);
+  return result;
+}
+
+Tensor remainder(const Tensor& self, const Tensor& other) {
+  Tensor result = at::empty_like(self);
+  return native::remainder_out(result, self, other);
```

Comment:
You might want to change `native` to `at` here and below for `remainder`. See #25689

Reply:
What about the other functions, e.g. `native::sub_out`, `native::atan2_out`?

Reply:
As far as I understood, using `native::`, some would cause issues but some would not. It is always safer to use `at::`. Could @ailzhang confirm this?

Reply:
@xuhdev Thanks for pinging! I think the ones in this diff might be fine, main reason being these functions all have device dispatch in `native_functions.yaml` (or if they have an entry in derivatives.yaml). 
Btw please let me know if the XLA test fail on this PR. (I think it shouldn't tho) ;)

Reply:
Thanks @ailzhang and @xuhdev. My build is failing when I use `at::`. I will try with `native::`.